### PR TITLE
Release 0.91 doc update

### PR DESCRIPTION
**Pull request by nermiller**
Updated files and links for 0.91 version

### CODE REVIEW

File: layouts/partials/flex/body-beforecontent.html [hunk 1]
```diff
@@ -35,50 +35,49 @@
...
+                    <span class="menu__version-selector__toggler closer version-selector-control">&#x25B2;</span>
+                </button>
+                <div class="menu__version-selector__list version-selector-control">
+                    <a href="https://docs.redis.com/6.0/platforms" id="version-select-6.x" onclick="_setSelectedVersion('6.0', 'v6.x (latest)')">v6.x (latest)</a>
+                    <a href="https://docs.redis.com/5.6/platforms" id="version-select-5.x" onclick="_setSelectedVersion('5.6', 'v5.x')">v5.x</a>
+                </div>
             </div>
-          {{end}}
-        {{end}}
+            {{end}}
```
Comment (nermiller) [inline layouts/partials/flex/body-beforecontent.html:69]:
```suggestion
{{end}}

```

**Comment by yaronp68** [comment]
The images of RDI docs are now in a folder without any sub folders and so they are all broken

File: layouts/partials/home/options.html [hunk 2]
```diff
@@ -103,28 +100,53 @@ <h5>Modules</h5>
...
             </a>
             </div>
             <hr>
             <ul>
                 <section>
-                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a
-                            href="/kubernetes/architecture">Architecture</a></li>
-                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a
-                            href="/kubernetes/rec/">Clusters</a></li>
+                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi">Quick Start</a></li>
```
Comment (rrelledge) [inline layouts/partials/home/options.html:116]: The quick start link on the home page does not take you to the quick start.
```suggestion
                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi/quickstart">Quick start</a></li>
```

File: layouts/partials/home/options.html [hunk 3]
```diff
@@ -103,28 +100,53 @@ <h5>Modules</h5>
...
+                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi/Architecture">Architecture</a></li>
```
Comment (rrelledge) [inline layouts/partials/home/options.html:117]: The architecture link on the home page leads to a 404.
```suggestion
                    <li><i class="fa fa-angle-right fa-lg category-icon"></i><a href="/rdi/architecture">Architecture</a></li>
```

File: content/rdi/installation/install-redis-gears.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+Title: Install RedisGears for Redis Data Integration
+linkTitle: RedisGears installation
+description: Install and set up RedisGears for a Redis Data Integration deplyoment.
```
Comment (rrelledge) [inline content/rdi/installation/install-redis-gears.md:4]:
```suggestion
description: Install and set up RedisGears for a Redis Data Integration deployment.
```

File: content/rdi/installation/debezium-server-configuration.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+- The property can include entries for multiple tables. Use a semicolon to separate table entries in the list:
+
+  ```properties
+  debezium.source.message.key.columns=<databaseName>.<tableName1>:<columnName1>,<columnName2>;<databaseName>:<tableName>:<columnName1>,<columnName2>
+  ```
+
+### Fully-qualified table name
+
+In this document we refer to the fully-qualified table name as `<databaseName>.<tableName>`. This format is for MySQL database. For Oracle, SQLServer and Postgresql databases use `<schemaName>`.`<tableName>` instead.
```
Comment (rrelledge) [inline content/rdi/installation/debezium-server-configuration.md:222]:
```suggestion
In this document we refer to the fully-qualified table name as `<databaseName>.<tableName>`. This format is for MySQL database. For Oracle, SQLServer, and PostgreSQL databases, use `<schemaName>`.`<tableName>` instead.
```

File: content/rdi/troubleshoot.md [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+  - Verify you have access to the specified ip:port `telnet <IP ADDRESS OF SERVER PC> <PORT>`.
+- **Using a Redis Enterprise user with the wrong role**
+
+  User must have roles of either `DB Member`, `Cluster Member` or `Admin`.
+
+- **Trying to connect to the Redis Data Integration DB without an Ingress Controller path**
+
+  When running Redis Enterprise on Kubernetes, each Redis DB has a service that is created only after the DB had been created. In order to connect with the CLI from outside of Kubernetes and configure the Redis Data Integration DB, you need an `Ingress` resource for your Redis Data Integration database and `Ingress Controller` running on the Kubernetes cluster.
+
+  A Kubernetes `Service` and `Ingress` are mutually exclusive. So if your database has a `Service` you should remove it and add an `Ingress` resource. Read [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/ingress/) for more information.
```
Comment (rrelledge) [inline content/rdi/troubleshoot.md:59]: Suggested a comma here. I'm also not sure about using "so" to start the second sentence. Maybe we can just remove it, replace it with another word, or rework these sentences a bit.
```suggestion
  A Kubernetes `Service` and `Ingress` are mutually exclusive. So if your database has a `Service`, you should remove it and add an `Ingress` resource. Read [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/ingress/) for more information.
```

File: content/rdi/troubleshoot.md [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+  **Alternatively, Just run the CLI withing the Kubernetes cluster as described [here](../installation/install-k8s.md).**
```
Comment (rrelledge) [inline content/rdi/troubleshoot.md:60]:
```suggestion
  **Alternatively, you can run the CLI within the Kubernetes cluster as described [here](../installation/install-k8s.md).**
```
Or maybe:
```suggestion
  **Alternatively, you can run the CLI on the Kubernetes cluster as described [here](../installation/install-k8s.md).**
```

File: content/rdi/data-transformation/_index.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+alwaysopen: false
+categories: ["redis-di"]
+aliases: 
+---
+
+The key functionality that RDI performs is mapping the data coming from [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html) (representing a Source Database row data or row state change) into a Redis key with a value of [Hash](https://redis.io/docs/data-types/hashes/) or [JSON](https://redis.io/docs/stack/json/).
+There are two types of Data Transformations in RDI:
+
+- By default, each source row is converted into one [Hash](https://redis.io/docs/data-types/hashes/) or one JSON key in Redis.
+  This conversion is using the Debezium Schema based conversion: the incoming data includes the schema and RDI will use a set of handlers to automatically convert each source column to a the a Redis Hash field or JSON type based on the Debezium type in the schema. [Look here](../reference/data-types-conversion/data-types-conversion.md) for a full reference of these conversions.
```
Comment (rrelledge) [inline content/rdi/data-transformation/_index.md:15]: Typo fix and a few other suggestions. Also maybe we can rework the "Look here" sentence a bit.
```suggestion
  This conversion uses the Debezium schema-based conversion: the incoming data includes the schema and RDI uses a set of handlers to automatically convert each source column to a Redis Hash field or JSON type based on the Debezium type in the schema. [Look here](../reference/data-types-conversion/data-types-conversion.md) for a full reference of these conversions.
```

File: content/rdi/quickstart/write-behind-guide.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+- Renaming a field
+- Adding a field
+- Removing a field
+- Mapping source fields to use in output
+
+To learn more about transformations, read [here](data-transformation/data-transformation-pipeline.md).
+
+## How to Provide Target's Secrets
+
+Target's secrets (e.g. TLS certificate) can be read from a path on the Redis Nodes file system. This allows to consume secrets injected from secret stores.
```
Comment (rrelledge) [inline content/rdi/quickstart/write-behind-guide.md:175]: It felt like there was a missing word in the second sentence. I just left a possible alternative suggestion to consider. Also, I wonder if "on the Redis Nodes file system" should be changed slightly to something like: "on the Redis node's file system."
```suggestion
The target's secrets (such as TLS certificates) can be read from a path on the Redis Nodes file system. This allows the consumption of secrets injected from secret stores.
```

File: content/rdi/quickstart/write-behind-guide.md [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+
+## Deploying the Write Behind Pipeline
+
+Starting the pipeline is done by running the [`deploy`](reference/cli/redis-di-deploy.md) command:
```
Comment (rrelledge) [inline content/rdi/quickstart/write-behind-guide.md:179]:
```suggestion
To start the pipeline, run the [`deploy`]({{<relref "/rdi/reference/cli/redis-di-deploy">}}) command:
```

File: content/rdi/quickstart/write-behind-guide.md [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+
+```bash
+redis-di deploy
+```
+
+You can check that the pipeline is running, receiving and writing data, using the [`status`](reference/cli/redis-di-status.md) command:
```
Comment (rrelledge) [inline content/rdi/quickstart/write-behind-guide.md:185]:
```suggestion
You can check that the pipeline is running, receiving, and writing data, using the [`status`]({{<relref "/rdi/reference/cli/redis-di-status">}}) command:
```

File: content/rdi/quickstart/write-behind-guide.md [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+
+## Upgrading
+
+If you need to upgrade RDI, you should use the [`upgrade`](reference/cli/redis-di-upgrade.md) command that provides zero downtime upgrade:
+
+```bash
+redis-di upgrade ...
+```
+
+Read [here](upgrade/upgrade-guide.md) about upgrade.
```
Comment (rrelledge) [inline content/rdi/quickstart/write-behind-guide.md:224]:
```suggestion
See the [upgrade guide]({{<relref "/rdi/upgrade">}}) for more information.
```

File: content/rdi/architecture.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+## Scalability and high availability
+
+Redis Data Integration is highly available:
+
+- At the feeder level: Debezium Server is deployed using [Kubernetes](https://kubernetes.io/) or [Pacemaker](https://clusterlabs.org/pacemaker/) for failover between stateless instances, while the state is secured in Redis.
+- At the Data and Control Plane: Using Redis Enterprise mechanisms for high availability (shard replica, cluster level configurations, etc.)
+
+Redis Data Integration is horizontally scalable:
+
+- Table level data streams can be split to multiple shards where they can be transformed and written to the target in parallel, while keeping table level order of updates.
```
Comment (rrelledge) [inline content/rdi/architecture.md:88]:
```suggestion
- Table-level data streams can be split to multiple shards where they can be transformed and written to the target in parallel, while keeping the table-level order of updates.
```

File: content/rdi/architecture.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+## Tested topologies
+
+Redis Data Integration is running on Redis Enterprise. It works on any installation of Redis Enterprise regardless of the runtime (bare metal, VMs or containers).
```
Comment (rrelledge) [inline content/rdi/architecture.md:92]: It felt like there was a missing word after "runtime".
```suggestion
Redis Data Integration runs on Redis Enterprise. It works on any installation of Redis Enterprise regardless of the runtime environment (bare metal, VMs, or containers).
```

File: content/rdi/architecture.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+Redis Data Integration can be collocated with the target Redis DB or run on a different Redis Enterprise cluster.
+
+Debezium Server should run in warm High Availability topology in one of the following options:
```
Comment (rrelledge) [inline content/rdi/architecture.md:95]: I'm not sure about "run in...topology" here, but I'm also not entirely sure what wording would fit best here. Also, I'm also not sure what "warm" means in this context.
```suggestion
Debezium Server should run as a warm high availability topology in one of the following options:
```

File: content/rdi/architecture.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+
+- Running on 2 VMs in an active/passive setup. The failover orchestration can be done by tools such as [Pacemaker](https://clusterlabs.org/pacemaker/doc/).
```
Comment (rrelledge) [inline content/rdi/architecture.md:97]:
```suggestion
- Running on 2 VMs in an Active-Passive setup. The failover orchestration can be done by tools such as [Pacemaker](https://clusterlabs.org/pacemaker/doc/).
```

File: content/rdi/architecture.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+- Running as a Kubernetes pod where Kubernetes watches the pod liveliness and readiness and acts to recover nonfunctional pod
```
Comment (rrelledge) [inline content/rdi/architecture.md:98]:
```suggestion
- Running as a Kubernetes pod where Kubernetes watches the pod liveliness and readiness and acts to recover nonfunctional pods.
```

File: content/rdi/architecture.md [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+  It is important to note that Redis Data Integration does not keep any state on Debezium. All state is kept in Redis Data Integration.
```
Comment (rrelledge) [inline content/rdi/architecture.md:99]: Maybe?
```suggestion

{{<note>}}
Redis Data Integration does not keep any states on Debezium. All states are kept in Redis Data Integration.
{{</note>}}
```

File: content/rdi/upgrade.md [hunk 19]
```diff
@@ -0,0 +1,95 @@
...
+
+```bash
+redis-di upgrade
+```
+
+## Upgrade RedisGears 
+
+RedisGears can be upgraded for a Redis Enterprise Cluster running Redis Data Integration without downtime. Please read the following section for exact instructions.
+
+**Important**: Failing to follow these steps in the right order, will result with the Redis Data Integration still running the old RedisGears version.
```
Comment (rrelledge) [inline content/rdi/upgrade.md:29]:
```suggestion
**Important**: If you do not follow these steps in the right order, Redis Data Integration will continue running the old version of RedisGears.
```

File: content/rdi/upgrade.md [hunk 20]
```diff
@@ -0,0 +1,95 @@
...
+
+Upgrading the RDI Engine is done using the RDI CLI. This action will upgrade the version of the RDI Engine to be the same one currently used by the running RDI CLI (only when the CLI version is higher than the RDI Engine version).
+
+```bash
+redis-di upgrade
+```
+
+## Upgrade RedisGears 
+
+RedisGears can be upgraded for a Redis Enterprise Cluster running Redis Data Integration without downtime. Please read the following section for exact instructions.
```
Comment (rrelledge) [inline content/rdi/upgrade.md:27]:
```suggestion
You can upgrade RedisGears for a Redis Enterprise cluster running Redis Data Integration without downtime. See the following section for detailed instructions.
```

File: content/rdi/upgrade.md [hunk 21]
```diff
@@ -0,0 +1,95 @@
...
+
+**Important**: Failing to follow these steps in the right order, will result with the Redis Data Integration still running the old RedisGears version.
+
+In order to upgrade the RedisGears module once a new release is available, the following steps should be followed:
```
Comment (rrelledge) [inline content/rdi/upgrade.md:31]:
```suggestion
To upgrade the RedisGears module after a new release is available, follow these steps:
```

File: content/rdi/upgrade.md [hunk 22]
```diff
@@ -0,0 +1,95 @@
...
+
+- Download the new version of RedisGears (Python package only):
```
Comment (rrelledge) [inline content/rdi/upgrade.md:33]: Maybe these instructions should be a numbered list instead.
```suggestion
- Download the new version of RedisGears (Python package only):
```

File: content/rdi/upgrade.md [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+
+  ```bash
+  curl -s https://redismodules.s3.amazonaws.com/redisgears/snapshots/redisgears_python.Linux-ubuntu18.04-x86_64.master.zip -o /tmp/redisgears.zip
+  ```
+
+- Install it in the cluster:
```
Comment (rrelledge) [inline content/rdi/upgrade.md:39]:
```suggestion
- Install the new RedisGears package in the cluster:
```

File: content/rdi/upgrade.md [hunk 24]
```diff
@@ -0,0 +1,95 @@
...
+
+  ```bash
+  curl -v -k -s -u "<CLUSTER_USER>:<CLUSTER_PASSWORD>" -F "module=@/tmp/redisgears.zip" https://<CLUSTER_HOST>:<CLUSTER_API_PORT>/v2/modules
+  ```
+
+- The above endpoint returns an action ID that you can monitor to see the installation progress by invoking this endpoint:
```
Comment (rrelledge) [inline content/rdi/upgrade.md:45]:
```suggestion
- To monitor installation progress, copy the action ID from the previous step's output and use it to replace `${action_uid}` in the following REST API request:
```

File: content/rdi/upgrade.md [hunk 25]
```diff
@@ -0,0 +1,95 @@
...
+
+  ```bash
+  curl -k -s -u "<CLUSTER_USER>:<CLUSTER_PASSWORD>" https://<CLUSTER_HOST>:<CLUSTER_API_PORT>/v1/actions/${action_uid}
+  ```
+
+  Once the status returned by this endpoint is `completed`, the new module is installed in the cluster.
+
+- Verify that the new version is installed via the Admin Console (settings → redis modules):
```
Comment (rrelledge) [inline content/rdi/upgrade.md:53]:
```suggestion
- Verify that the new version is installed via the admin console (**settings > redis modules**):
```

File: content/rdi/upgrade.md [hunk 26]
```diff
@@ -0,0 +1,95 @@
...
+
+ ![Cluster Redis modules](/images/rdi/cluster-redis-modules.png)
+
+
+ - From the Admin console, go to your Redis Data Integration configuration view (databases → Redis Data Integration → configuration), you’ll see an indication that there’s an update available to your RedisGears module:
```
Comment (rrelledge) [inline content/rdi/upgrade.md:58]:
```suggestion
 - From the admin console, go to your Redis Data Integration configuration view (**databases > Redis Data Integration > configuration**). You should see an indication that there’s an update available for your RedisGears module:
```

File: content/rdi/upgrade.md [hunk 27]
```diff
@@ -0,0 +1,95 @@
...
+
+  ![Redis Data Integration update available](/images/rdi/redis-di-db-update-available.png)
+  
+  You can also see this indication via the `rladmin`:
```
Comment (rrelledge) [inline content/rdi/upgrade.md:62]:
```suggestion
  You can also see this indication using the `rladmin status` command:
```

File: content/rdi/upgrade.md [hunk 28]
```diff
@@ -0,0 +1,95 @@
...
+  active - SMUpgradeBDB init
+  active - SMUpgradeBDB restart
+  .completed - SMUpgradeBDB
+  Done
+  ```
+
+  Notes:
+
+  - Replace the rg version with your version (can be taken from the json available in the zip file of the module, see [instructions]({{<relref "/modules/install/upgrade-module">}}).
+  - It's important to specify `keep_args` argument so the configuration won’t be reset.
```
Comment (rrelledge) [inline content/rdi/upgrade.md:89]:
```suggestion
  - It's important to specify the `keep_args` argument so the configuration won’t be reset.
```

File: content/rdi/upgrade.md [hunk 29]
```diff
@@ -0,0 +1,95 @@
...
+  - Warning - After you upgrade the module for a database, the database shards restart. This causes a short interruption in the availability of this database across the cluster.
+
+- Verify that the new RedisGears is used by Redis Data Integration by navigating to **databases > Redis Data Integration > configuration**:
```
Comment (rrelledge) [inline content/rdi/upgrade.md:92]:
```suggestion
- Verify that Redis Data Integration is using the new version of RedisGears by navigating to **databases > Redis Data Integration > configuration**:
```

**Review by rrelledge — COMMENTED** [review]
Here are some more suggestions.

**Pull request merged**